code/API_definitions/CAMARA/number_verification.yaml

```diff
@@ -278,11 +281,11 @@ components:
       content:
         application/json:
           schema:
-            $ref: '#/components/schemas/NotAuthenticatedByIpPermissionDenied'
+            $ref: '#/components/schemas/PhoneNumberVerificationPermissionDenied'
```

Comment:
The more generic error descriptions are there, the better, so this change is good.